CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ See also https://docs.resim.ai/changelog/ for all ReSim changes
 
 ### Unreleased
 
+### v0.1.23 - September 29 2023
+
+#### Added
+
+- The `experiences` command now has:
+  - `tag` and `untag` subcommands for tagging and untagging experiences
+  - A `list` subcommand for listing experiences
+
+- There is a new `experience-tags` command with the following subcommands:
+  - `create`
+  - `list-experiences` for listing the experiences with a given tag
```

Comment:
Should we add a `list` to this as well?

Reply:
added https://app.asana.com/0/1205272835002601/1205604302612464/f